### ISSUE TITLE
revert - Browser support update

### ISSUE THREAD

**Issue by hakantr** [mdn/browser-compat-data#7037]
[revert](https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/revert)

Hi, the browser support needs to be updated for the `revert` value.

**Comment by sideshowbarker** [comment]
Thanks much for reporting this. I’ve raised https://github.com/mdn/browser-compat-data/pull/7038 with an update for the data.

**Comment by karelbilek** [comment]
Still shows as unsupported in MDN one month later. Is there some other issue?

**Comment by karelbilek** [comment]
Oh. I made an empty newline edit and it re-rendered the compat table.

Strange. Anyway it helped.

**Comment by sideshowbarker** [comment]
> Oh. I made an empty newline edit and it re-rendered the compat table.

Yeah, sorry the details about that aren’t super-discoverable, but do have them documented at https://github.com/mdn/browser-compat-data/blob/master/docs/contributing.md#updating-compatibility-tables-on-mdn.

As explained there, if you’re logged into your MDN account and do a force-reload when viewing any article, that’ll cause the CMS to regenerate the article contents — which includes pulling in any updated browser-compat data.

So in other words, you don’t actually even need to do this empty-edit thing — just a force-reload will do the trick.

**Comment by karelbilek** [comment]
Oh OK.

Sorry about commenting on both the PR and the issue.

**Comment by sideshowbarker** [comment]
> Sorry about commenting on both the PR and the issue.

 No worries. It was right for you to ask — it’s a good reminder that we should maybe try to find a way to make it more clear in general to all contributors and people who take time to raise issues
